docs/source/ide_basics.rst

```diff
@@ -0,0 +1,131 @@
+IDE File Basics
+===============
+
+What's an IDE file?
+-------------------
+
+An IDE file is a read-only hierarchical data format that stores recording information generated by an enDAQ sensor device. It contains both time-indexed data from several different kinds of sensors (like acceleration, pressure, temperature, etc.), as well as metadata about the recording device (like device serial number, model number, device name, etc.) and recording settings.
+
+
+.. _dataset_desc:
+
+Accessing an IDE file
+---------------------
+
+The top-level interface for an IDE file is the :class:`Dataset` object, through which one can access all of the above-listed information. When you open a file for reading, for example, this is the type of object that is returned.
+
+Opening an IDE File
+~~~~~~~~~~~~~~~~~~~
+
+You can open an IDE file like so::
+
+    filename = "/your/file/path.IDE"
+    with idelib.importFile(filename) as ds:
+        print(type(ds))
+
+Note: a :class:`Dataset` object perfoms *lazy-loading*, meaning that it only loads information as is needed. As a result, it internally retains a handle to the source file which after use needs to be closed. This can be accomplished by either using :class:`Dataset` as a *context manager* (as seen above; this is the recommended method), or by using :class:`Dataset` as a normal value and calling the ``close()`` method manually::
+
+    filename = "C:\\your\\file\\path.IDE"
```

Comment:
There are a few Windows paths in this file; personally, I think generic cross-platform paths are better for documentation, even if it's what Windows would show (like `ds.filename` below).

Reply:
well "cross-platform" would mean something like `os.path.join("your", "file", "path")`. I assume you meant `"your/file/path"`, right?

Reply:
(partially?) addressed in 65a79aa993e7c14739a4064c41538991e9bdd2ed

Reply:
The latter. `os.path.join()` would be best practice if the path were potentially going to be used _outside_ of Python (e.g., shell execution or passing arguments to a native OS component).

I suspect various path delimiters (`\\`, possibly `:`) might work on different platforms the same way `/` will work in Windows, but sticking to standard `/` is simpler.

Reply:
aure, I'm fine w/ that. changed in 65a79aa993e7c14739a4064c41538991e9bdd2ed